tests/vmi_configuration_test.go

```diff
@@ -624,8 +681,10 @@ var _ = Describe("[sig-compute]Configurations", func() {
 		Context("[rfe_id:140][crit:medium][vendor:cnv-qe@redhat.com][level:component]with diverging memory limit from memory request and no guest memory", func() {
 			It("[test_id:3115]should show the memory limit inside the VMI", func() {
 				vmi := libvmi.NewCirros()
-				vmi.Spec.Domain.Resources.Limits = kubev1.ResourceList{
-					kubev1.ResourceMemory: resource.MustParse("256Mi"),
+				vmi.Spec.Domain.Resources = v1.ResourceRequirements{
```

Comment:
This is overriding the memory, do we want it? Let's change the name if we do.

Reply:
Thanks, yeah.
Changed